src/backend/templates/schedule.hbs

```diff
@@ -561,6 +590,19 @@
         exportICS();
       });
 
+      $('.dropdown-submenu a').on("click", function(e){
+        $(this).next('ul').toggle();
+        if($(this).find('span').css('display') === 'none'){
+        $(this).find('span').attr("style","display:inline-flex");
+        }
+        else{
+        $(this).find('span').attr("style","display:none");
+        }
+        $(this).parent().siblings().eq(0).children(".dropdown-inner").attr("style","display:none");
+        $(this).parent().siblings().eq(0).children("a").find('span').attr("style","display:none");
```

Comment:
You could consider these attempts to reduce the code a little:

- You can use `is(':visible')` as a more robust alternative to `.css('display') === 'none'`
- You can use `css()` instead of `attr('style', ...)`
- You can use `hide()` instead of setting `display: none`

```js
       // Check the span's old status
       // if (!$(this).find('span').is(':visible')) {

       // Or is it better to stay in sync with the new status of the <ul>?
       if ($(this).next('ul').is(':visible')) {
         $(this).find('span').css(display: 'inline-flex');
       } else {
         $(this).find('span').css(display: 'none');
       }

       $(this).parent().siblings().eq(0).children(".dropdown-inner").hide();
       $(this).parent().siblings().eq(0).children("a").find('span').hide();
```